invenio_rdm_records/services/pids/providers/datacite.py

```diff
@@ -24,6 +25,25 @@ def __init__(self, username, password, prefix, test_mode, **kwards):
         self.test_mode = test_mode
 
 
+class DataCiteClientFromConfig(DataCiteClient):
+    """DataCite Client.
+
+    It Loads the values from config.
+    """
+
+    def __init__(self, name, **kwards):
+        """Constructor."""
+        self.name = name
+        name = self.name.upper()
+
+        super(DataCiteClientFromConfig, self).__init__(
+            username=current_app.config[f"{name}_DATACITE_CLIENT_USERNAME"],
+            password=current_app.config[f"{name}_DATACITE_CLIENT_PASSWORD"],
+            prefix=current_app.config[f"{name}_DATACITE_CLIENT_PREFIX"],
+            test_mode=current_app.config[f"{name}_DATACITE_CLIENT_TEST_MODE"],
```

Comment:
Ideally `test_mode` does not depend on the specific account on the DataCite, but honestly I kind of like it here... :)

Reply:
I think is the easiet way to state it... since we need to put the rest of the variables in config... even if not changed much, it's a nice to have there just for consistency.

Reply:
IRL, instead of `[]`, do get and default to `True`